akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -52,8 +58,17 @@ object CircuitBreaker {
    * @param callTimeout [[scala.concurrent.duration.FiniteDuration]] of time after which to consider a call a failure
    * @param resetTimeout [[scala.concurrent.duration.FiniteDuration]] of time after which to attempt to close the circuit
    */
-  def create(scheduler: Scheduler, maxFailures: Int, callTimeout: FiniteDuration, resetTimeout: FiniteDuration): CircuitBreaker =
+  def create(
+    scheduler:    Scheduler,
+    maxFailures:  Int,
+    callTimeout:  FiniteDuration,
+    resetTimeout: FiniteDuration): CircuitBreaker =
     apply(scheduler, maxFailures, callTimeout, resetTimeout)
+
+  def exceptionAsFailure[T]: Try[T] ⇒ Boolean = {
```

Comment:
I think simply 

`Bifunction<T, Throwable, Boolean> will do?`

As it's java, the user should be aware to catch null if they expect null